From a3eb6ba425773224076c41aabc3c490a6a016ee6 Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Thu, 18 Aug 2022 14:31:27 +0200
Subject: [PATCH] mm: remove libaio from being skipped

Since libaio is now used also in critical section,
keep the libaio locked in memory.
---
 lib/mm/memlock.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/mm/memlock.c b/lib/mm/memlock.c
index 3d1a3927ceb4..f43bacdcdf27 100644
--- a/lib/mm/memlock.c
+++ b/lib/mm/memlock.c
@@ -105,7 +105,6 @@ static const char * const _blacklist_maps[] = {
 	"/LC_MESSAGES/",
 	"gconv/gconv-modules.cache",
 	"/ld-2.",		/* not using dlopen,dlsym during mlock */
-	"/libaio.so.",		/* not using aio during mlock */
 	"/libattr.so.",		/* not using during mlock (udev) */
 	"/libblkid.so.",	/* not using blkid during mlock (udev) */
 	"/libbz2.so.",		/* not using during mlock (udev) */
-- 
2.26.2

